html/forms/html-form-structure/payment-form.html

```diff
@@ -41,21 +41,21 @@ <h2>Contact information</h2>
                 <span>Name: </span>
                 <strong><span aria-label="required">*</span></strong>
```

Comment:
On all of these, should it be `aria-label="required"` or `aria-required="true"`? I suspect the former but will check.

Reply:
The answer I got is that this is good.

> the aria-required=true would be on the form control. it isn’t necessary if the required attribute is set and valid, but can be used on a component or element that is required but isn’t a form control.

So we don't need it because we're setting required on the fields.

> The aria-label="required"  is appropriate for the *, as it labels the asterisk, providing an appropriate accessible name to the character that means “required”  but wouldn’t otherwise mean that.
Best practices are to include a visible comment for ALL users explaining what the asterisk means.

Reply:
Ok